src/simulators/extended_stabilizer/extended_stabilizer_state.hpp

```diff
@@ -777,6 +802,38 @@ void State::statevector_snapshot(const Operations::Op &op, ExperimentResult &res
      BaseState::qreg_.statevector(norm_estimation_samples_, 3, rng));
 }
 
+double State::expval_pauli(const reg_t &qubits,
```

Comment:
Here we should add the `rng` argument to the function, so that the seed_simulator value is respected.

We can either remove `expval_pauli` from base class, or you can just implement a trivial instance of the base class function `expval_pauli(args) {return 0; }` and only use the rng version of the function